pkg/console/console.go

```diff
@@ -126,46 +127,55 @@ func generateConsoleCSPString(config *Config, nonce string) string {
 	return webui.GenerateCSPString(cspMap)
 }
 
+func cspHeader(console *Console) webmiddleware.MiddlewareFunc {
```

Comment:
We need to decide if we keep those in the function or extract them. SInce we don't reuse, I don't think it makes much sense to rip them out of the function.